src/app.py

```diff
@@ -107,5 +107,27 @@ def add_course():
     return redirect(url_for("start"))
 
 
+# Router to course detailed page
+@app.route("/course_page", methods=["GET", "POST"])
+def course_page():
+    df = get_df_from_csv_in_s3(s3, bucket_name, mock_data_file)
+    courses = df.loc[0, "courses"]  # For PoC purpose
+    # Parsing it into a Python list
+    courses = ast.literal_eval(courses)
+    return render_template("course_page.html", courses=courses)
+
+# Router to study plan detailed page
+@app.route("/plan_page", methods=["GET", "POST"])
+def plan_page():
+    return render_template("plan_page.html")
+
+# Router to user profile page
+@app.route("/profile_page", methods=["GET", "POST"])
+def profile_page():
+    df = get_df_from_csv_in_s3(s3, bucket_name, mock_data_file)
+    username = df.loc[0, "username"]
```

Comment:
Similar to what I commented above, username is also a global variable so we could remove these 2 lines and can directly access it